configure.ac

```diff
@@ -1222,6 +1222,7 @@ if test "$ZEND_MAINTAINER_ZTS" = "yes"; then
   PHP_THREAD_SAFETY=yes
 else
   PHP_THREAD_SAFETY=no
+  LIBS="$LIBS -pthread"
```

Comment:
I'm quite sure this needs more work ... probably copy the flag detection routine from TSRM ?